lib/media-queries.js

```diff
@@ -0,0 +1,43 @@
+
+var _ = require('lodash')
+var postcss = require('postcss')
+var rules = require('./rules')
+var selectors = require('./selectors')
+var declarations = require('./declarations')
+
+module.exports = function (root, opts) {
+
+  var result = {
+    total: 0,
+    unique: 0,
+    values: [],
+    contents: [] // Not sure what to name this
```

Comment:
Also, with this proposed implementation, are media query stats also making into the overall stats for the CSS? I'm failing to see where that is occurring currently.